tasks/parallel_verification.py

```diff
@@ -147,6 +145,22 @@ def run_impl(
             extra=logging_context,
         )
 
+        is_success = top_level_totals_match and file_level_totals_match
+        sentry_sdk.metrics.incr(
+            "parallel_verification.comparisons",
+            tags={"result": "success" if is_success else "failure"},
+        )
+        if not is_success:
+            with sentry_sdk.new_scope() as scope:
+                if file_level_mismatched_files:
+                    scope.set_extra("mismatched_files", file_level_mismatched_files)
+                if not top_level_totals_match:
+                    scope.set_extra("parallel_totals", parallel_report.totals.astuple())
+                    scope.set_extra("serial_totals", serial_report.totals.astuple())
+                sentry_sdk.capture_message(
```

Comment:
Where does this sort of thing show up on the Sentry UI? Would this be considered an error?

Reply:
It shows up in the issues list, but it is not considered an error.